### PR TITLE
New metaldust properties

### PR DESCRIPTION
**Pull request by james-trayford**
See also https://github.com/SWIFTSIM/pipeline-configs/pull/180, enabling new multi-size grain dust properties and extra particle properties to be read in.

### CODE REVIEW

File: velociraptor/autoplotter/compare.py [hunk 1]
```diff
@@ -65,6 +65,13 @@ def load_yaml_line_data(
     for path, name in zip(paths, names):
         try:
             with open(path, "r") as handle:
+                inc = 0
```
Comment (bwvdnbro) [inline velociraptor/autoplotter/compare.py:68]: What exactly is the point of this bit of code? Is this a leftover from debugging?

File: velociraptor/autoplotter/objects.py [hunk 2]
```diff
@@ -1160,7 +1161,7 @@ def create_plots(
                     _, _, exc_traceback = sys.exc_info()
                     print("Traceback:")
                     traceback.print_tb(exc_traceback, limit=10, file=sys.stdout)
-            except UnitConversionError as e:
+            except (UnitConversionError, ConversionError) as e:
```
Comment (bwvdnbro) [inline velociraptor/autoplotter/objects.py:1164]: Is there any particular reason why a `ConversionError` would be thrown by the new data?

**Review by bwvdnbro — CHANGES_REQUESTED** [review]
I'm happy with the changes in `registration.py`, but I don't understand the changes in the other files. Are they necessary? If so, could you please add some comments explaining why these changes/additions have been made?

**Comment by james-trayford** [comment]
@bwvdnbro you're right these other file changes are obsolete, updated

**Review by bwvdnbro — APPROVED** [review]

**Comment by bwvdnbro** [comment]
Could you apply the (brand new) formatting script before I merge this?

**Comment by james-trayford** [comment]
@bwvdnbro thanks, hope that's all good

**Pull request merged**